src/admin/menu/class-bem-menu-walker.php

```diff
@@ -120,7 +122,7 @@ public function start_el( &$output, $item, $depth = 0, $args = array(), $id = 0
     $parent_class = $prefix . $suffix['parent_item'];
 
     if ( ! empty( $item->classes ) ) {
-      $user_classes = array();
+      $user_classes = [];
```

Comment:
The below `foreach` can be refactored with `array_map` 🙂

Reply:
this will be refactored by Tihomir so I will leave it for him